src/main/java/frc/robot/commands/AutoIntakeCommand.java

```diff
@@ -0,0 +1,32 @@
+package frc.robot.commands;
+
+import edu.wpi.first.math.filter.Debouncer;
+import edu.wpi.first.math.filter.Debouncer.DebounceType;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.CrocodileSubsystem;
+
+public class AutoIntakeCommand extends CommandBase{
+    private final CrocodileSubsystem crocodileSubsystem;
+    private final double speed;
+    public AutoIntakeCommand(CrocodileSubsystem crocodileSubsystem, double speed){
+        this.crocodileSubsystem = crocodileSubsystem;
+        this.speed = speed;
+        addRequirements(crocodileSubsystem);
+    }
+    @Override
+    public void initialize() {
+        crocodileSubsystem.setIntakeSpeed(speed);
+    }
+    @Override
+    public boolean isFinished() {
+        Debouncer debouncer = new Debouncer(0.5, DebounceType.kBoth);
```

Comment:
This will construct a new debouncer every time `isFinished()` is called, meaning that as i understand it, you will always calculate it to false, which I don't think is what you want. Try constructing the debouncer in the constructor of the command instead.

Reply:
thanks, nice catch!